domain/policy_config.go

```diff
@@ -0,0 +1,34 @@
+package domain
+
+type PolicyTargetApplication struct {
```

Comment:
I really dont like this name becuase I feel like it can easily be confused with policy targets. May be `ConfigTargetApplication`....same for `PolicyTargetResource` below

Reply:
or `ConfigMatchApplication` and `ConfigMatchResource` to be more aligned with the yaml